features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,337 @@
+
+  De verblijfplaatsen worden aflopend gesorteerd geleverd. De eerste verblijfplaats in de verblijfplaatsen lijst is de meest recente verblijfplaats in de gevraagde periode.
+  Komt er in de gevraagde periode alleen verblijfplaatsen voor met bekend datum aanvang adreshouding/adres buitenland, dan is de sortering gelijk aan het aflopend sorteren op datum aanvang adreshouding/adres buitenland.
+
+  Abstract Scenario: <scenario> (geen vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | <datum van>         |
+    | datumTot            | <datum tot>         |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat | datumAanvangAdreshouding |
+    | 0800010000000001                 | Laan   | 20100818                 |
+
+    Voorbeelden:
+    | datum van  | datum tot  | scenario                                                                             |
+    | 2010-09-01 | 2011-09-01 | De gevraagde periode ligt na datum aanvang adreshouding                              |
+    | 2010-01-01 | 2010-08-19 | datumTot van de gevraagde periode is gelijk aan de dag na datum aanvang adreshouding |
+    # 24 |--A1--     |--A1--
+    #      |---| |---|
+
+  Abstract Scenario: <scenario> (wel vorige en geen volgende verblijfplaats)
+    Gegeven de persoon met burgerservicenummer '000000024' is ingeschreven op adres 'A1' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20100818                           |
+    En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+    | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+    | 0800                              | 20160526                           |
+    Als gba verblijfplaatsen wordt gezocht met de volgende parameters
+    | naam                | waarde              |
+    | type                | RaadpleegMetPeriode |
+    | burgerservicenummer | 000000024           |
+    | datumVan            | 2016-09-01          |
+    | datumTot            | 2020-01-01          |
+    Dan heeft de response verblijfplaatsen met de volgende gegevens
+    | adresseerbaarObjectIdentificatie | straat      | datumAanvangAdreshouding |
+    | 0800010000000002                 | Luttestraat | 20160526                 |
+
+    Voorbeelden:
+    | datum van  | scenario                                                  |
```

Comment:
Dit is een abstract scenario, maar de <datum van> wordt nergens gebruikt.

Reply:
aangepast